.github/workflows/semver-label-check.yml

```
@@ -0,0 +1,21 @@
+name: "Label Checker"
+on:
+  pull_request:
+    types: [opened, labeled, unlabeled, synchronize]
+
+jobs:
+  check_labels:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check for required labels
+        uses: actions/github-script@v6
```

actionlint (1.7.11)

[error] the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)